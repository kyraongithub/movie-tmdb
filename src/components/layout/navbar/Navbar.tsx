import React from 'react';

const Navbar: React.FC = () => {
  return (
    <nav className="bg-gray-900 text-white flex items-center justify-between p-4">
      <p className="text-2xl">st‚≠êr movie</p>

      <div className="flex gap-5">
        <a className="hover:scale-110 transition-all" href="/">
          Movie List
        </a>
        <a className="hover:scale-110 transition-all" href="/explore">
          Explore
        </a>
      </div>
    </nav>
  );
};

export default Navbar;
