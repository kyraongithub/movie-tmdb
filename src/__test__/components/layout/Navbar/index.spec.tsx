import { render } from '@testing-library/react';
import Navbar from '../../../../components/layout/navbar/Navbar';

describe('Navbar', () => {
  it('render Navbar component', () => {
    const { container } = render(<Navbar />);
    expect(container).toMatchSnapshot();
  });
});
