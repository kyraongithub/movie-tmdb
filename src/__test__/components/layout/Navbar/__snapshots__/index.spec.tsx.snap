// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Navbar > render Navbar component 1`] = `
<div>
  <nav
    class="bg-gray-900 text-white flex items-center justify-between p-4"
  >
    <p
      class="text-2xl"
    >
      st‚≠êr movie
    </p>
    <div
      class="flex gap-5"
    >
      <a
        class="hover:scale-110 transition-all"
        href="/"
      >
        Movie List
      </a>
      <a
        class="hover:scale-110 transition-all"
        href="/explore"
      >
        Explore
      </a>
    </div>
  </nav>
</div>
`;
